Relax version requirements for TensorFlow v2.5.0 to match the Python environemnt in EB
author: Alex Domingo (Vrije Universiteit Brussel)
--- tensorflow/tools/pip_package/setup.py.orig	2021-11-25 10:06:05.914491576 +0100
+++ tensorflow/tools/pip_package/setup.py	2021-11-25 10:08:48.414821521 +0100
@@ -78,19 +78,19 @@
     # NOTE: As numpy has releases that break semver guarantees and several other
     # deps depend on numpy without an upper bound, we must install numpy before
     # everything else.
-    'numpy ~= 1.19.2',
+    'numpy >= 1.19.2',
     # Install other dependencies
     'absl-py ~= 0.10',
     'astunparse ~= 1.6.3',
-    'flatbuffers ~= 1.12.0',
+    'flatbuffers >= 1.12.0',
     'google_pasta ~= 0.2',
-    'h5py ~= 3.1.0',
+    'h5py >= 3.1.0',
     'keras_preprocessing ~= 1.1.2',
     'opt_einsum ~= 3.3.0',
     'protobuf >= 3.9.2',
-    'six ~= 1.15.0',
+    'six >= 1.15.0',
     'termcolor ~= 1.1.0',
-    'typing_extensions ~= 3.7.4',
+    'typing_extensions >= 3.7.4',
     'wheel ~= 0.35',
     'wrapt ~= 1.12.1',
     # These packages need to be pinned exactly as newer versions are
